dash/dash-config.cmake.in

```diff
@@ -8,14 +8,14 @@
 
 @PACKAGE_INIT@
 
-set(DASH_VERSION_MAJOR "@DASH_VERSION_MAJOR@")
-set(DASH_VERSION_MINOR "@DASH_VERSION_MINOR@")
-set(DASH_VERSION_PATCH "@DASH_VERSION_PATCH@")
+set(DASH_VERSION_MAJOR "@DASH_VERSION_MAJOR@")
+set(DASH_VERSION_MINOR "@DASH_VERSION_MINOR@")
+set(DASH_VERSION_PATCH "@DASH_VERSION_PATCH@")
 
 set(DASH_LIBRARY "@DASH_LIBRARY@")
 set(DASH_LIBRARIES ${DASH_LIBRARY} "dart-@dart_variant@")
 set(DASH_CXX_FLAGS "@VARIANT_ADDITIONAL_COMPILE_FLAGS@ @CMAKE_CXX_FLAGS_RELEASE@")
```

Comment:
No way. Its not actually clear to me, how these cmake files should be used at all. While `find_package` works, I still need ugly ways to extract the flags from the package and add them to my target. I would have expected, that a simple `add_dependency(my-dash-app dash-mpi)` suffice, but its not. See [here](https://github.com/mephisto-hpc/testing/blob/master/CMakeLists.txt#L65) for my current approach.